chromatic/imports.py

```diff
@@ -36,6 +36,9 @@ def custom_formatwarning(message, *args, **kwargs):
 # For modelling transits.
 import batman
 
+# For converting Rainbows to pandas dataframe
+import pandas as pd
```

Comment:
I'm adding `pandas` to the list of `install_requires` in `setup.py` to make sure that it gets installed with `chromatic`. It seems very reasonable to include given that it's so fundamental.